script/export-components

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# A script to export Flynn components to a TUF repository.
+#
+# PREREQUISITES:
+#
+# - Install go-tuf
+#   go get github.com/flynn/go-tuf/cmd/tuf
+#
+# - Set the TUF passphrases
+#   export TUF_TARGETS_PASSPHRASE=xxxxxx
+#   export TUF_SNAPSHOT_PASSPHRASE=xxxxxx
+#   export TUF_TIMESTAMP_PASSPHRASE=xxxxxx
+
+set -eo pipefail
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+source "${ROOT}/script/lib/ui.sh"
+
+usage() {
+  echo "usage: $0 TUF_DIR" >&2
+}
+
+main() {
+  if [[ $# -ne 1 ]]; then
+    usage
+    exit 1
+  fi
+
+  local tuf_dir=$1
+
+  cd "${tuf_dir}"
+  tuf clean
+  cp "${ROOT}/host/bin/flynn-host"         "staged/targets/flynn-host"
+  cp "${ROOT}/host/bin/flynn-init"         "staged/targets/flynn-init"
+  cp "${ROOT}/cli/flynn-cli"               "staged/targets/flynn"
```

Comment:
This needs an OS/architecture in the filename, as we'll want to have others in the same repo the future.